README.md

```diff
@@ -0,0 +1,65 @@
+# Codacy Chart
+
+This is the top level chart to run Codacy in enterprise mode.
+With this chart, it is  possible to run all the components from Codacy,
+plugs all the dependencies with a single command line, including Codacy,
+PostgreSQL, RabbitMQ, Minio, Ingress etc...
+
+## Goals
+
+Core goals for this project:
+
+1. Easy to scale pods horizontally
+2. Easy to deploy, upgrade, maintain
+3. Wide support of cloud service providers
+4. Use external data stores, such as Minio (S3) or Amazon RDS (Postgres), whenever possible
+5. Users should be able to use their own certificates or use Let's Encrypt
+6. Users should be able to use a load balancer or an ingess
+
+We will try to leverage standard Kubernetes features:
+
+* ConfigMaps for managing configuration. These will then get mapped or passed to
+Docker containers
+* Secrets for sensitive data
```

Comment:
I would remove this. I think it is more related to planning than the actual finished chart.